### ISSUE TITLE
Failing the "Trying to unfollow #1"

### ISSUE THREAD

**Issue by mdrzn** [instabot-py/instabot.py#1207]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [x] I have searched for other issues with the same problem. (including #1153)

## Hosting
- [x] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ ] Bug
- [ ] Feature Request
- [x] Technical Help
- [ ] Other

### Operating System? (include version)
Linux

## Python Version?
3

## Description
I set it to follow 150 people a day and unfollow 100. It manages to like and follow with no issues, but when it's time to unfollow it gets stuck:

```
17.04.2018_11:09 Trying to unfollow #1:
17.04.2018_11:09 Getting user info : _redacted_
```

and then just moves on to the next iteration of likes:

```
17.04.2018_11:14 Trying to like media: 1759294814469062401
17.04.2018_11:14 Liked: 1759294814469062401. Like #7.
17.04.2018_11:14 Trying to comment: 1759240721704942782
17.04.2018_11:14 Write: "Nice!". #2.
17.04.2018_11:14 Trying to unfollow #1:
17.04.2018_11:14 Getting user info : _redacted_
17.04.2018_11:15 Trying to like media: 1759240721704942782
17.04.2018_11:15 Liked: 1759240721704942782. Like #8.
```

This are the settings:
```
 follow_per_day=50,
 follow_time=1 * 60,
 unfollow_per_day=350,
 unfollow_break_min=25,
 unfollow_break_max=35,
 log_mod=0,
```

**Comment by JorgeLekk** [comment]
same

**Comment by zhen737** [comment]
Try https://github.com/instabot-py/instabot.py/issues/1153#issuecomment-382123394 (This should fixed unfollow mode 0)

**Comment by mdrzn** [comment]
Using this fixed it: [instabot.py-master.zip](https://github.com/instabot-py/instabot.py/files/1912308/instabot.py-master.zip)

Just paste your old example.py in there and you're good to go

**Comment by zhen737** [comment]
@mdrzn Is your following database "follows_db.db" on the right place?

**Comment by massy384** [comment]
It work for me. Thank you guys !

**Comment by mdrzn** [comment]
@zhen737 I just unzipped the new master and replaced the example.py with my own configuration, left everything else in place. 

I now checked again on the script, in about 9 hours it only unfollowed 18 people. Sometimes it works, sometimes it doesn't. Like it starts "Trying to unfollow #9:", gets the whole info on him, then just skips to the next step without actually unfollowing.

**Comment by constantinosbou** [comment]
Tried your solution @zhen737 but i get ''Looks like there is nobody to unfollow''.

**Comment by Blademontane** [comment]
I downloaded the new file, but it mae zero difference. It still does not get the User Info, and just tries to unfollow another user, but caznnot because it cannot get the users info. I even tried the fix mentioned in #1218, but still does not work.
